Dockerfile.prometheus

```diff
@@ -0,0 +1,5 @@
+from centos/ruby-25-centos7
+workdir /app
+add . .
+add entrypoint.prometheus.sh /app/
+expose 9394
```

Comment:
This really isn't necessary. The following works great in the docker-compose (there are other changes not shown - I need to submit a PR to update the docker-compose.yml, but here is the [Dockerfile](https://github.com/akofink/compliance-backend/blob/prometheus-with-docker/Dockerfile) and [docker-compose.yml](https://github.com/akofink/compliance-backend/blob/prometheus-with-docker/docker-compose.yml) for now):

```yml
  prometheus:
    image: compliance-backend-rails
    restart: on-failure
    volumes:
      - .:/app:z
    entrypoint: bash -c 'bundle exec prometheus_exporter -c lib/graphql_collector.rb'
```